docs/reference/api/docker_remote_api_v1.20.md

```diff
@@ -268,6 +269,7 @@ Json Parameters:
           Specified as a boolean value.
     -   **Dns** - A list of DNS servers for the container to use.
     -   **DnsSearch** - A list of DNS search domains
+    -   **DnsOptions** - A list of DNS options
```

Comment:
Changes in API version v1.20 should now be moved to v1.21 (which is the version to be used in Docker 1.9)